src/daemon/daemon_rpc.cpp

```diff
@@ -59,6 +62,11 @@ auto make_server(const std::string& server_address, multipass::Rpc::Service* ser
     if (server == nullptr)
         throw std::runtime_error("Failed to start multipass gRPC service at " + server_address);
 
+    auto group = getgrnam("adm");
+    auto socket_path = QString::fromStdString(server_address).section("unix:", 1, 1).toStdString();
+    chown(socket_path.c_str(), 0, group->gr_gid);
+    chmod(socket_path.c_str(), S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP);
```

Comment:
We need to check the return value here... if we failed to change the permissions of the socket, it's pretty much useless, so I would consider this a fatal condition - it needs to throw a std::runtime_error